vaadin-combo-box-flow-parent/vaadin-combo-box-testbench/src/main/java/com/vaadin/flow/component/combobox/testbench/MultiSelectComboBoxElement.java

```diff
@@ -129,6 +129,14 @@ public void deselectByText(String label) {
         executeScript(script, this, label);
     }
 
+    /**
+     * Deselects all items, effectively clearing the value.
+     */
+    public void deselectAll() {
```

Comment:
Added convenience method for deselecting all values.

Another option would have been to override `clear`, however it didn't seem appropriate for clearing the selection, as the JavaDoc mentions that it's supposed to clear input elements specifically.